nixos/modules/services/system/kerberos/kerberos-server.md

```diff
@@ -0,0 +1,55 @@
+# kerberos_server {#module-services-kerberos-server}
+
+Kerberos is a computer-network authentication protocol that works on the basis of tickets to allow nodes communicating over a non-secure network to prove their identity to one another in a secure manner.
+
+This module provides both the MIT and Heimdal implementations of the a Kerberos server.
+
+## Usage {#module-services-kerberos-server-usage}
+
+To enable a Kerberos server:
+
+```nix
+{
+  security.krb5 = {
+    # Here you can choose between the MIT and Heimdal implementations.
+    package = pkgs.krb5;
+    # package = pkgs.heimdal;
+
+    # Optionally set up a client on the same machine as the server
+    enable = true;
+    settings = {
+      libdefaults.default_realm = "EXAMPLE.COM";
+      realms."EXAMPLE.COM" = {
+        kdc = "kerberos.example.com";
+        admin_server = "kerberos.example.com";
+      };
+    };
+  }
+
+  services.kerberos-server = {
+    enable = true;
+    settings = {
+      realms."EXAMPLE.COM" = {
+        acl = [{ principal = "adminuser"; access=  ["add" "cpw"]; }];
+      };
+    };
+  };
+}
+```
+
+## Notes {#module-services-kerberos-server-notes}
+
+- The Heimdal documentation will sometimes assume that state is stored in `/var/heimdal`, but this module uses `/var/lib/heimdal` instead.
```

Comment:
Would it be a good idea to patch the docs in the package to point to /var/lib/heimdal?

Reply:
I think this could be more effort than reward. While I like the idea of having the manpages corrected to satisfy the distro's quirks, the rest of the docs assumes a lot of other stuff that is not true for NixOS as well that is nontrivial to change and maintain. I think most users will already read the docs with a grain of salt.

Do you propose patching the path with a repo-wide find and replace?

Reply:
Ah, looks like all the man page files are all over the repo. That's annoying. I assumed they were all in the doc directory and then blanket replace in just that directory with the two paths would have been fine, but like this it would be pretty sketchy to do imo.

Yeah, then leave it as is for now. I opened an issue in their repo: https://github.com/heimdal/heimdal/issues/1244